### PR TITLE
Select channel

### PR DESCRIPTION
**Pull request by iceeye7gabi**
Front End pentru partea de canale. Momentan se vad doar canalele care pot sa exista. Nu am buton de login
Este facut dupa un tutorial de pe net.
Tule-n gura de fisiere XML.

### CODE REVIEW

File: SelectChannel/src/main/java/com/morse/selectchannel/SelectChannelActivity.java [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+import android.widget.AdapterView;
+import android.widget.ArrayAdapter;
+import android.widget.ImageView;
+import android.widget.ListView;
+import android.widget.TextView;
+import android.widget.Toast;
+
+public class SelectChannelActivity extends AppCompatActivity {
+    ListView listView;
+    String mTitle[] = {"Messenger", "Whatsapp", "Twitter", "Instagram"};
```
Comment (raduitache) [inline SelectChannel/src/main/java/com/morse/selectchannel/SelectChannelActivity.java]: We don't have an API for WhatsApp, except for the business version. We do not have the funds nor the actual motivation to invest money into that. So, no WhatsApp

File: SelectChannel/src/main/java/com/morse/selectchannel/SelectChannelActivity.java [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+    String mDescription[] = {"Canalul pentru Messenger", "Canalul pentru Whatsapp", "Canalul pentru Twitter", "Canalul pentru Instagram"};
```
Comment (raduitache) [inline SelectChannel/src/main/java/com/morse/selectchannel/SelectChannelActivity.java]: Change these to English, for consistency.

File: SelectChannel/src/main/java/com/morse/selectchannel/SelectChannelActivity.java [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+                }
+                if (position ==  0) {
+                    Toast.makeText(SelectChannelActivity.this, "Canalul pentru Whatsapp", Toast.LENGTH_SHORT).show();
+                }
+                if (position ==  0) {
+                    Toast.makeText(SelectChannelActivity.this, "Canalul pentru Twitter", Toast.LENGTH_SHORT).show();
+                }
+                if (position ==  0) {
+                    Toast.makeText(SelectChannelActivity.this, "Canalul pentru Instagram", Toast.LENGTH_SHORT).show();
+                }
```
Comment (raduitache) [inline SelectChannel/src/main/java/com/morse/selectchannel/SelectChannelActivity.java]: You made a list with these strings. Why not use those?

File: SelectChannel/src/main/java/com/morse/selectchannel/SelectChannelActivity.java [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+
+            // now set our resources on views
+            images.setImageResource(rImgs[position]);
+            myTitle.setText(rTitle[position]);
+            myDescription.setText(rDescription[position]);
+            return row;
+        }
+    }
+
+}
```
Comment (raduitache) [inline SelectChannel/src/main/java/com/morse/selectchannel/SelectChannelActivity.java]: Shouldn't this have its own file?

File: SelectChannel/src/main/res/drawable/ic_launcher_background.xml [hunk 5]
```diff
@@ -0,0 +1,170 @@
...
+        android:fillColor="#00000000"
+        android:pathData="M69,19L69,89"
+        android:strokeWidth="0.8"
+        android:strokeColor="#33FFFFFF" />
+    <path
+        android:fillColor="#00000000"
+        android:pathData="M79,19L79,89"
+        android:strokeWidth="0.8"
+        android:strokeColor="#33FFFFFF" />
+</vector>
```
Comment (raduitache) [inline SelectChannel/src/main/res/drawable/ic_launcher_background.xml:170]: I have no idea what these are but ok.

**Review by HristeaIonut — CHANGES_REQUESTED** [review]
Signal instead of Whatsapp

File: SelectChannel/src/main/res/values/strings.xml [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+        "Components with responsive elevations may encounter other components as they move between "
+        "their resting elevations and dynamic elevation offsets. Because material cannot pass "
+        "through other material, components avoid interfering with one another any number of ways, "
+        "whether on a per component basis or using the entire app layout.\n"
+        "On a component level, components can move or be removed before they cause interference. "
+        "For example, a floating action button (FAB) can disappear or move off screen before a "
+        "user picks up a card, or it can move if a snackbar appears.\n"
+        "On the layout level, design your app layout to minimize opportunities for interference. "
+        "For example, position the FAB to one side of stream of a cards so the FAB won’t interfere "
+        "when a user tries to pick up one of cards.\n\n"
```
Comment (raduitache) [inline SelectChannel/src/main/res/values/strings.xml]: Pretty cool, not gonna lie

**Review by raduitache — CHANGES_REQUESTED** [review]

File: SelectChannel/src/main/java/com/morse/selectchannel/SelectChannelActivity.java (hunk 3 above)
Comment (HristeaIonut) [inline SelectChannel/src/main/java/com/morse/selectchannel/SelectChannelActivity.java]: positions should be == 0, 1, 2 or 3

**Review by HristeaIonut — CHANGES_REQUESTED** [review]

File: SelectChannel/src/main/res/values/strings.xml (hunk 6 above)
Comment (iceeye7gabi) [reply SelectChannel/src/main/res/values/strings.xml]: I erased everything from here. It was from the default module page.

File: SelectChannel/src/main/res/drawable/ic_launcher_background.xml (hunk 5 above)
Comment (iceeye7gabi) [reply SelectChannel/src/main/res/drawable/ic_launcher_background.xml:170]: Me too...

File: SelectChannel/src/main/java/com/morse/selectchannel/SelectChannelActivity.java (hunk 4 above)
Comment (iceeye7gabi) [reply SelectChannel/src/main/java/com/morse/selectchannel/SelectChannelActivity.java]: Yes, I created a separate file for this.

File: SelectChannel/src/main/java/com/morse/selectchannel/SelectChannelActivity.java (hunk 3 above)
Comment (iceeye7gabi) [reply SelectChannel/src/main/java/com/morse/selectchannel/SelectChannelActivity.java]: I used them now.

File: SelectChannel/src/main/java/com/morse/selectchannel/SelectChannelActivity.java (hunk 2 above)
Comment (iceeye7gabi) [reply SelectChannel/src/main/java/com/morse/selectchannel/SelectChannelActivity.java]: Yep, did it.

File: SelectChannel/src/main/java/com/morse/selectchannel/SelectChannelActivity.java (hunk 1 above)
Comment (iceeye7gabi) [reply SelectChannel/src/main/java/com/morse/selectchannel/SelectChannelActivity.java]: Yes, I modified.

File: SelectChannel/src/main/java/com/morse/selectchannel/SelectChannelActivity.java (hunk 3 above)
Comment (iceeye7gabi) [reply SelectChannel/src/main/java/com/morse/selectchannel/SelectChannelActivity.java]: Yep , I did it. Now the buttons should work as intended.

**Review by HristeaIonut — APPROVED** [review]
🙂

**Review by raduitache — APPROVED** [review]

**Pull request merged**